### PR TITLE
Skipto update and removing jumpto.js

### PR DESCRIPTION
**Pull request by jongund**
This pull request:

1. Fixes SkipTo.js bug related to Safari rendering of menu items, which was including scrollbars, fixed by using `overflow-y: clip`.
2. Removes jumpto.js, since no longer being used
3. Updated `app.js` to add `scripto.js`, instead of `jumpto.js`.
4. NOTE: The behavior SkipTo is set to "popup", which is what is being used on the [ARIA Practices](https://www.w3.org/WAI/ARIA/apg/) website.  This means the "Skip To Content" button is not visible on page load, you must tab on to the page to see the button. 

I would be interested in learning more about how the `skipto.js` script gets translated and configured (if any) for the APG website.

Jon

___
[WAI Preview Link](https://deploy-preview-159--wai-aria-practices2.netlify.app)

### CODE REVIEW

**Review by alflennik — COMMENTED** [review]
Thank you for simplifying the examples and upgrading the older jumpto implementation to the newer skipto implementation. I was able to verify that the issues mentioned in the PR are fixed, and I tested Chrome, Firefox and Safari on Mac and Chrome, Firefox and Edge on Windows. There are some visual differences caused by the configuration being different, and I think I'd vote that we preserve the existing behavior, but I'll leave the decision of how to proceed up to others who might have more context. Thanks!

**Comment by alflennik** [comment]
The skipto configuration has `displayOption` set to `popup` instead of `static` - just for continuity I think my preference would be to keep it as `static`.

You asked in the PR description how this file is processed when deployed to the website, and what happens is it does a find and replace on this file to change the `displayOption` to `popup`.

**Comment by alflennik** [comment]
I reran the test several times to make sure the reason the CI is failing is not a flaky test, but it does fail consistently. In the `menubar_menubar-navigation` there is an error that seems related to this change:

```
Sending key "ENTER" to menuitem referencing menu with id="id-skip-to-menu-50" in menubar should display submenu
```

It seems that the HTML ID may have changed due to upgrading the library.

**Comment by alflennik** [comment]
I just pushed fixes for the issues I identified above. Since I'm contributing to this PR now I assigned @howard-e to help me review. Thanks!

**Comment by mcking65** [comment]
@alflennik 

Thank you for the fixes. Is it yet possible to get a netlify preview?

File: examples/js/skipto.js [hunk 1]
```diff
@@ -118,7 +118,7 @@
         buttonBackgroundColor: '#ddd',
```
[the remark this answers is not in the document]
Comment (nschonni) [reply examples/js/skipto.js:123]: Probably something to raise at https://github.com/paypal/skipto, since the file is vendored from there

**Comment by alflennik** [comment]
@mcking65 the preview can be viewed here: https://github.com/w3c/wai-aria-practices/pull/158. I created a PR on the WAI-ARIA Practices Repo because there were some lingering references to jumpto that prevented the preview link from building. It's probably worth noting that the current published version of the APG is already using skipto.

**Comment by howard-e** [comment]
> Is it yet possible to get a netlify preview?

@mcking65 @alflennik re-ran the action for triggering the WAI Preview Link being generated for this PR given that https://github.com/w3c/wai-aria-practices/pull/158 has been approved and merged. The link is now attached.

**Review by howard-e — APPROVED** [review]
LGTM

**Comment by shirsha** [comment]
There is no change in the appearance of the[ skip to link in the preview site](https://deploy-preview-159--wai-aria-practices2.netlify.app/aria/apg/).

image

**Review by shirsha — APPROVED** [review]
Skip to link appears the same in production and preview site

**Pull request merged**